src/translations/i18n.js

```diff
@@ -1,12 +1,15 @@
 
 const i18n = {
-  nb: {
-    numberToWord: (tall) => {
-      const ord = ['to', 'tre', 'fire', 'fem', 'seks', 'sju', 'åtte', 'ni', 'ti', 'elleve', 'tolv'];
-      return tall > 12 ? tall : ord[tall - 2];
+    nb: {
```

Comment:
Kjører du `eslint src/js/**/*.js src/__tests__/**/*.js --no-ignore` for å fikse lintingen i denne fila?

Reply:
For meg sier eslint at det bare er LF/CRLF feil